subprojects/docs/src/docs/userguide/java_gradle_plugin.adoc

```diff
@@ -61,4 +61,5 @@ For each plugin you are developing, add an entry to the `gradlePlugin {}` script
 The `gradlePlugin {}` block defines the plugins being built by the project including the `id` and `implementationClass` of the plugin. From this data about the plugins being developed, Gradle can automatically:
 
 * Generate the plugin descriptor in the `jar` file's `META-INF` directory.
-* Configure the <<publishing_maven,Maven>> or <<publishing_ivy,Ivy>> publishing plugins to publish a <<sec:plugin_markers,Plugin Marker Artifact>> for each plugin.
+* Configure the <<publishing_maven,Maven>> or <<publishing_ivy,Ivy Publish Plugins>> to publish a <<sec:plugin_markers,Plugin Marker Artifact>> for each plugin.
```

Comment:
The line below is another good reason to add a `guidesUrl` attribute to the build.